rwengine/include/rw_mingw.hpp

```diff
@@ -0,0 +1,3 @@
+#undef IGNORE
+#undef near
+#undef far
```

Comment:
What do these collide with? Could needing these undefinitions be avoided by including different windows headers or by changing the name that conflict?